carts/views.py

```diff
@@ -1,3 +1,83 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+from django.db.models import Sum
+
+from users.models    import User
+from products.models import Product, ProductGroup
+from .models         import Cart
+from core.utils      import signin_decorator
+
+class CartListView(View):
+    @signin_decorator
+    def post(self, request):
+        try :
+            data = json.loads(request.body)
+            product_id = data['ProductId']
+            quantity   = data['quantity']
+            color_id   = data['ColorId']
+            user_id    = request.user.id
+
+            cart, created = Cart.objects.get_or_create(
+                product_id = product_id,
+                color_id   = color_id,
+                user_id    = user_id
+            )
+            if created:
+                cart.quantity = 0
+            cart.quantity +=quantity
+            cart.save()
```

Comment:
cart의 quantity 컬럼에 default value를 0으로 주시면 아래 코드가 필요 없을 것 같네요?

```suggestion
            cart.quantity +=quantity
            cart.save()
```